app/root/root.tsx

```diff
@@ -16,7 +16,7 @@ import UserPreferences from "../preferences/preferences";
 declare var window: any;
 
 interface State {
-  user: User;
+  user?: User;
```

Comment:
Seems like the tests are failing -- I wonder if we have some logic somewhere that is checking specifically for `user === null`. Might be less risky to change this to `user: User | null`

Reply:
Per discussion it's this code in invocation.tsx: https://github.com/buildbuddy-io/buildbuddy/blob/e753ffa527b2fb96b4095d4ee6acb29c32dfbc0f/app/invocation/invocation.tsx#L152

But I flipped it back to "| null" and added a todo to change it to optional in the future.